pkg/controller/plan/validation_test.go

```diff
@@ -95,6 +108,62 @@ var _ = ginkgo.Describe("Plan Validations", func() {
 			// secret should NOT be set on plan.Referenced.Secret
 			gomega.Expect(plan.Referenced.Secret).To(gomega.BeNil())
 		})
+
+		ginkgo.DescribeTable("should validate plan-level PVCNameTemplate",
+			func(template string, shouldBeValid bool) {
+				source := createProvider(sourceName, sourceNamespace, "", v1beta1.OpenShift, &core.ObjectReference{})
+				destination := createProvider(destName, destNamespace, "", v1beta1.OpenShift, &core.ObjectReference{})
+				plan := createPlan(testPlanName, testNamespace, source, destination)
+
+				// Set the plan-level PVC name template
+				plan.Spec.PVCNameTemplate = template
+
+				source.Status.Conditions.SetCondition(condition.Condition{Type: condition.Ready, Status: condition.True})
+				destination.Status.Conditions.SetCondition(condition.Condition{Type: condition.Ready, Status: condition.True})
+
+				reconciler = createFakeReconciler(plan, source, destination)
```

Comment:
I see that the `createFakeReconciler` is missing the secret 

unlike the 

```
			reconciler = createFakeReconciler(secret, plan, source, destination)
```

Reply:
THANKS !